src/Data/HugeNum.purs

```diff
@@ -73,6 +74,14 @@ timesSign Plus Plus = Plus
 timesSign Minus Minus = Plus
 timesSign _ _ = Minus
 
+instance isForeignHugeNum :: IsForeign HugeNum where
+  read f = do
+    n <- readNumber f
+    pure (fromNumber n)
+
+instance asForeignHugeNum :: AsForeign HugeNum where
+  write v = write (toNumber v)
```

Comment:
Roger that. Sounds good. It's good to have this discussion here as documentation, as I'm sure others will be curious about this. I'll close - thanks Gary!